front/public/locales/fr/stdcm.json

```diff
@@ -24,12 +25,15 @@
   "stdcmResults": "Résultats",
   "stdcmSimulationReport": "Fiche simulation",
   "trainPath": {
+    "addVia": "Ajouter un PR intermédiaire",
```

Comment:
Are we sure we want to use abbreviations ?

Reply:
![image](https://github.com/OpenRailAssociation/osrd/assets/103027832/5eb58b98-3289-4f62-904c-25186a4a6052)
That is what is asked on the mockup but we can see with @thibautsailly if that is something to change ?